tests/testthat/test_PipeOpCor.R

```diff
@@ -0,0 +1,40 @@
+test_that("PipeOpCor - basic properties", {
+  pop = po("fda.cor")
+  expect_pipeop(pop)
+  expect_identical(pop$id, "fda.cor")
+})
+
+test_that("PipeOpCor works", {
+  set.seed(1234L)
```

Comment:
`withr::local_seed()` to not make all tests deterministic

Reply:
more than happy to add, but how is the effect different from explicitly setting the seed in each test, since it just resets the seed after the scope?